pom.xml

```diff
@@ -60,6 +56,42 @@
           </instructions>
 	</configuration>
       </plugin>
+	  
+		 <plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-compiler-plugin</artifactId>
+			<version>3.8.0</version>
+			<dependencies>
+				<dependency>
+					<groupId>org.ow2.asm</groupId>
```

Comment:
Quick question: why is this needed? It is not a direct dependency of `maven-compiler-plugin`, but I assume comes transitively from somewhere? I don't mind it being there, just want to understand.
I think shade plugin at least depends on it?

Reply:
3.8.0 was released a bit before ASM, ASM 7.0 is the jpms approved one (also guice 4.2.2 etc). Came across this one while doing cglib/asm constructs.

I didn't want to put it into dependency management though, so I kept it local to the compiler plugin.